### ISSUE TITLE
Local State Variable Rename

### ISSUE THREAD

**Issue by strongishdeveloper** [FlutterFlow/flutterflow-issues#505]
Issue tracker is **ONLY** used for reporting bugs. New feature suggestions and questions should be discussed on Community or submitted through our user feedback form.

Your issue may already be reported! Please search in the [issue tracker](../) before creating one.

Please **thumbs up** this issue if you have also experienced it. You may also add more information if there is something relevant that was not mentioned. However, please refrain from comments that are not constructive, like "I have this problem too", etc.

## Expected behavior (required)
Successful editing of the Field Name of a state variable

## Current behavior (required)
When you change a local page state variables name, the build fails because a reference to the old name of the variable is hanging around in the dart file generated by FF.

```
lib/edit_athlete_profile/edit_athlete_profile_widget.dart:587:62: Error: The setter 'aditingWeightclass' isn't defined for the class 'EditAthleteProfileModel'.
 - 'EditAthleteProfileModel' is from 'package:---/edit_athlete_profile/edit_athlete_profile_model.dart' ('lib/edit_athlete_profile/edit_athlete_profile_model.dart').
Try correcting the name to the name of an existing setter, or defining a setter or field named 'aditingWeightclass'.
                                                      _model.aditingWeightclass =
```

## To Reproduce (required)

Create a page with a local state variable
Set this variable through an action
Change the Fieldname of the variable
Build

### Bug Report Code (required, generate by right clicking inside Widget Tree and clicking "Get Bug Report Code")

IT48hc/1z89NocBE1qqBbMAxnjwWH0c7RK8o1tVHSQE8J5zKB+wAYvf7P1FWTLy5TX9PeGagkj8Jxvfqv+L+G/oCFEmbUb5nwqgMWgvzfGOmV6bROruJMS0ffaMKRSz+

## Context (required)

The build fails without explaination of the issue, when run through Android Studio I can then see it. 
_Current work around is to delete the variable, then make a new one, then update any action that might reference it_

## Your environment

* Version of FlutterFlow used: 3.1
* Platform (e.g. Web, MacOS Desktop): Web
* Browser name and version: Chrome - latest
* Operating system and version (desktop or mobile): Windows

**Comment by hariprasadms** [comment]
Hi @strongishdeveloper - Thanks for reporting this issue. I have tested and can reproduce the issue. 

Note - This issue mostly happening while updating local state names which are used in actions.

![image](https://user-images.githubusercontent.com/17435678/219949449-471eb6f4-1014-4d4e-8d76-4f4ba1eb99c9.png)

**Comment by bulgariamitko** [comment]
I don't think this is a real bug as dealing with custom code it is the user responsibility. 
@strongishdeveloper can you confirm that you are NOT using custom code and local state together?

**Comment by strongishdeveloper** [comment]
> I don't think this is a real bug as dealing with custom code it is the user responsibility. 
> @strongishdeveloper can you confirm that you are NOT using custom code and local state together? 

Yes, can confirm this isn't to do with custom code. The error happens on a clean project as well. 

It really isn't a big issue tbh. Just thought you'd like to know. The easy fix for users is to just delete the local state variable, then recreate it

**Comment by Kainkainkain** [comment]
I am also having this issue- I wanted to rename my page state variables to more appropriate names as my app got more complex and it threw errors. After trial and error, viewing code - I found this to be the issue: the model has the new name and the widget still uses the old in actions. I tried changing the action slightly to see if a new update would fix but no. 
Rather than try rebuilding the action (time-costly) I simply changed the model name (the app state variable name) back to the original, and will have to live with my (now-confusing) naming conventions.
